mirte_telemetrix/scripts/mappings/pcb.py

```diff
@@ -86,7 +59,70 @@ def get_max_pwm_value():
     return board_mapping.get_max_pwm_value()
 
 
+def generate_motor_mapping(pin_a, pin_b):
+    # pin a has preference for pwm
+    # This will make sure, dp, pp and dd is always possible when using a connector without knowing what type of control
+    #     	A	B
+    # pp	P1	P2
+    # dp	P1	D1
+    # dd	D2	D1 # Not available
```

Comment:
What do you mean with "# not available"?

Reply:
We only have pp, dp and ddp, not dd. We might add that or I'll remove this line